### ISSUE TITLE
Firefox Extention?

### ISSUE THREAD

**Issue by jlczuk** [gatewayapps/kamino#150]
Is there any chance of a firefox extension?

**Comment by johnmurphy01** [comment]
Of course! You're actually the first person to ask but it should be trivial to port this to a Firefox addon.

**Comment by jlczuk** [comment]
All I can say is, SUWEEET!  If I knew how to do it, I would fork and issue a PR.  Thanks!

**Comment by johnmurphy01** [comment]
After looking a little bit, appears there will be some work to do for the port. Definitely doable, but may take a little longer than I expected.

**Comment by johnmurphy01** [comment]
@all-contributors please add @jlczuk for ideas

**Comment by DavidGriffith** [comment]
How close is this effort to fruition?

**Comment by DavidGriffith** [comment]
> @DavidGriffith hasn't been started. Just based on my above comments, my guess is it would take a little bit of work but can likely be done with minimal changes. Firefox does now support Manifest v3 just like Chrome so it could be as easy as uploading to the Firefox extension store. I will just have to check and see

I'm hoping that this would provide a solution to https://github.com/orgs/community/discussions/67437.  I've tried a bunch of stuff that just don't work.

**Comment by johnmurphy01** [comment]
> > @DavidGriffith hasn't been started. Just based on my above comments, my guess is it would take a little bit of work but can likely be done with minimal changes. Firefox does now support Manifest v3 just like Chrome so it could be as easy as uploading to the Firefox extension store. I will just have to check and see
> 
> I'm hoping that this would provide a solution to https://github.com/orgs/community/discussions/67437. I've tried a bunch of stuff that just don't work.

Unfortunately, Kamino only works within github.com. The extension is only enabled for that domain and uses the github api to move stuff around. I have very little exposure to Gitlab but it's possible Kamino could fulfill that need, but there would need to be work to make all of that happen.

**Comment by DavidGriffith** [comment]
> > > @DavidGriffith hasn't been started. Just based on my above comments, my guess is it would take a little bit of work but can likely be done with minimal changes. Firefox does now support Manifest v3 just like Chrome so it could be as easy as uploading to the Firefox extension store. I will just have to check and see
> > 
> > 
> > I'm hoping that this would provide a solution to https://github.com/orgs/community/discussions/67437. I've tried a bunch of stuff that just don't work.
> 
> Unfortunately, Kamino only works within github.com. The extension is only enabled for that domain and uses the github api to move stuff around. I have very little exposure to Gitlab but it's possible Kamino could fulfill that need, but there would need to be work to make all of that happen.

My big problem is getting data into Github rather than getting it out of Gitlab.  I've been able to get that out of Gitlab in the form of CSV files, but getting it into Github is very troublesome.  In fact, I'm having a nasty time simply getting a CSV file from Github and putting it right back into different repo within Github.  I'm hoping to use this extension to do the second half of this task.

**Comment by johnmurphy01** [comment]
@DavidGriffith understood. Moving content into different repos is pretty painless with Kamino. I tried uploading the extension to the Firefox addon store yesterday. There were some validation issues so those will need to be fixed before Kamino can be available on Firefox. As I have time I will try to get these resolved
